hack/operatorhub/config.yaml

```
@@ -1,6 +1,6 @@
-newVersion: 1.9.0
-prevVersion: 1.8.0
-stackVersion: 7.16.0
+newVersion: 2.0.0
+prevVersion: 1.9.1
+stackVersion: 8.0.0
```

Just a thought. This file will stay like this in `main`, isn't it? I know we never did that before but we could already set the future versions.
```suggestion
newVersion: 2.1.0-SNAPSHOT
prevVersion: 2.0.0
stackVersion: 8.1.0-SNAPSHOT
```

Then when we cut a release branch, we just have to remove the `-SNAPSHOT` like in https://github.com/elastic/cloud-on-k8s/pull/5256.